WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -236,32 +259,93 @@ - (NSDictionary *)mapPostFormatsFromResponse:(id)response
 
 - (RemoteBlogSettings *)remoteBlogSettingFromJSONDictionary:(NSDictionary *)json
 {
-    NSDictionary *rawSettings = [json dictionaryForKey:BlogRemoteSettingsKey];
+    NSAssert([json isKindOfClass:[NSDictionary class]], @"Invalid Settings Kind");
     
-    RemoteBlogSettings *remoteSettings = [RemoteBlogSettings new];
+    RemoteBlogSettings *settings = [RemoteBlogSettings new];
+    NSDictionary *rawSettings = [json dictionaryForKey:RemoteBlogSettingsKey];
     
-    remoteSettings.name = [json stringForKey:BlogRemoteNameKey];
-    remoteSettings.desc = [json stringForKey:BlogRemoteDescriptionKey];
-    remoteSettings.defaultCategory = [rawSettings numberForKey:BlogRemoteDefaultCategoryKey] ?: @(BlogRemoteUncategorizedCategory);
+    // General
+    settings.name = [json stringForKey:RemoteBlogNameKey];
+    settings.tagline = [json stringForKey:RemoteBlogTaglineKey];
+    settings.privacy = [rawSettings numberForKey:RemoteBlogPrivacyKey];
```

Comment:
I'd recommend against using SafeExpectations for REST. Our APIs should deliver what they promise, and if they don't we want this to fail so we now about it.

https://developer.wordpress.com/2014/12/08/on-api-correctness/

If something isn't what it's supposed to be, report it, and then use SafeExpectations but leave a TODO/issue to remove it when the API is fixed.